common/protob/messages-management.proto

```diff
@@ -71,18 +71,19 @@ message Features {
         Capability_Binance = 3;
         Capability_Cardano = 4;
         Capability_Crypto = 5;                  // generic crypto operations for GPG, SSH, etc.
-        Capability_EOS = 6;
-        Capability_Ethereum = 7;
-        Capability_Lisk = 8;
-        Capability_Monero = 9;
-        Capability_NEM = 10;
-        Capability_Ripple = 11;
-        Capability_Stellar = 12;
-        Capability_Tezos = 13;
-        Capability_U2F = 14;
-        Capability_Shamir = 15;
-        Capability_ShamirGroups = 16;
-        Capability_PassphraseEntry = 17;        // the device is capable of passphrase entry directly on the device
+        Capability_Decred = 6;
+        Capability_EOS = 7;
+        Capability_Ethereum = 8;
+        Capability_Lisk = 9;
+        Capability_Monero = 10;
+        Capability_NEM = 11;
+        Capability_Ripple = 12;
+        Capability_Stellar = 13;
+        Capability_Tezos = 14;
+        Capability_U2F = 15;
+        Capability_Shamir = 16;
+        Capability_ShamirGroups = 17;
+        Capability_PassphraseEntry = 18;        // the device is capable of passphrase entry directly on the device
```

Comment:
The mappings between message IDs and currencies changed, is this backwards-compatible?